### DISCUSSION TITLE
How can i get filtering for each column in q-table

### DISCUSSION THREAD

**Discussion by Nbtank06** [quasarframework/quasar#10537] [Quasar App Extensions]
Hi I was going to ask you something. How can i get filtering for each column in q-table, quasar table column filtering similar to datatables?

**Comment by ontwikkelfabriek** [comment]
@Nbtank06 

Not in the default Quasar component Q-table. Here's an extension for Qv1 that has more features:
https://quasar-qgrid.netlify.app/examples
https://github.com/pratik227/quasar-qgrid

btw I would look for 3de party vue components that are more database orientated.

**Reply by Nbtank06** [reply]
Thank you  @ontwikkelfabriek

**Reply by oopsaleem** [reply]
I also need this feature @ontwikkelfabriek, please.

**Comment by sonicviz** [comment]
I've just come across this need. I've tried AG-Grid in Quasar, which has great column filters, but it doesn't seem you can use quasar components inside the cell rendered in AG-Grid, so i've gone back to Q-Table. 

QGrid doesn't appear to work with QTable. Are there any other solutions? 

Thanks!

**Comment by s-light** [comment]
https://github.com/pratik227/quasar-qgrid
seems promising.
